content/blog/high-utilization-in-systems/index.md

```diff
@@ -0,0 +1,114 @@
+
+## Why does this myth of high utilization persist?
+
+If doing things this way impacts delivery speed and quality in the long run, why do we still see high utilization as a positive signal? My theory is that most of our industry is caught up playing short term games. 
+
+Leadership is expected to provide results in short-term cycles, hence management gets recognized for delivering projects in tight timelines and developers are rewarded when they are able to ship tasks as fast as possible. None of these behaviours are responsible for creating better products, teams or culture in the long run. This reminds me of a quote from Dave Packard:
+
+> “More Organisations Die Of Indigestion Than Starvation”
+
+This quote really resonates with me. We need teams focused on a small number of objectives that are aligned with the company vision. High utilization doesn't allow for strategic planning, getting teams caught up in a tactical mindset.
+
+## Things to look out for
+
+If you are not sure whether your team is under high utilization or not (you probably know) you can look for the following characteristics:
+
+* Team members working alone or in small teams due too many initiatives
+* Poor engagement in meetings
+* Asynchronous channels (Slack, email, etc) drying up in participation or being slow/ignored
+* Pull request lead cycle taking a long time
+    - PRs are taking longer to be opened (high complexity/lower quality in the codebase)
+    - PRs are taking longer to be reviewed (context switching) 
+* Increased amount of bugs in production
+* Knowledge silos (small groups each holding part of the knowledge while no one in the team holds the full picture)
+* Lack of documentation
+
+Pay special attention to decisions on how the strategic objectives should be delivered being made without or far from the team that is executing it.
+
+## Caveats
+
+There are moments where high utilization is unavoidable. For example, if your company is trying to find market fit or when time to market is critical for success. These tend to be highly overestimated by companies, but it is a legitimate scenario that should be taken into consideration. With that said, keeping teams under high utilization for long periods of time is not a viable strategy.
+
+## Conclusion
+
+It takes a team to push against the myth of high utilization. Developers should raise their concerns when they feel they are lacking support or stretched too thin. Knowing when to say "no" is a powerful skill to have. Managers should listen to the team's feedback, but also keep an eye on and push back when they feel like the team is juggling between too many responsibilities. You will rarely hear a "no" from junior developers and it is your job to pay attention to a high utilization situation. Leadership on the other hand should trust that their teams are making good decisions on how their strategic objectives should be delivered.
```

Comment:
When an entire team is under high utilization, its different than if an individual is under high utilization. IE: If there is a junior dev saying yes to everything because they want to look good, its easy to then say No and have tasks given to others. However, if an entire team is under high utilization, its often the case that even if everyone wants to say no, someone has to do it. In cases like this maybe this blog post should touch on strategies as well.

If you are looking to add more advice for how to deal with utilization, one thing I tried to do was book time in my calendar for work. Ie: If I was working on something, and someone came to me with a request such as "can you review this PR", I would add in a 30-45min timeslot in my calendar to review. Example: 

<img width="623" alt="Screen Shot 2022-04-19 at 4 18 56 PM" src="https://user-images.githubusercontent.com/15018469/164088682-a8ccdbc5-fc44-43d9-a712-cc8031a49ea7.png">

<img width="579" alt="Screen Shot 2022-04-19 at 4 21 46 PM" src="https://user-images.githubusercontent.com/15018469/164089139-d3d86b7d-8652-4570-ba46-df80e3a77bad.png">

That way If I'm asked to do something, I can budget time for it. And say I see my next day is booked with tasks I set aside time for, then I can let the person know "I will get to this in 2 days". This was really helpful when under high utilization, as you can then focus on a task and not have that stress knowing you have 100 things to do (since you know time is coming eventually where you can dedicate focus to it).

Its also nice that when something high priority comes in, you can easily slide the timeblock of a lesser priority task to another day, and slot in the higher priority work.

Reply:
This is a great way to mitigate prioritization overload and context switching, and set expectations as a bonus. It's also a great way to raise visibility with the manager. 

> If there is a junior dev saying yes to everything because they want to look good, its easy to then say No and have tasks given to others. However, if an entire team is under high utilization, its often the case that even if everyone wants to say no, someone has to do it. 

You would be surprised. In many cases the manager can actually wait if you explain the reason why. Of course it only works if the manager is thinking long-term. If everything is a priority to the manager it becomes really hard to manage expectations with them. Ultimately the hope is that hearing from multiple people in the team that they don't have the bandwidth will force the manager to reprioritize.